client-app/src/examples/contact/mobile/AppModel.ts

```diff
@@ -50,17 +50,14 @@ export default class AppModel extends BaseAppModel {
 
     constructor() {
         super();
+        makeObservable(this);
```

Comment:
We need to add `makeObservable` here for the `tagList` and `contacts` observables above